packages/tools/fetch-tool/src/fluidFetchMessages.ts

```diff
@@ -205,7 +205,7 @@ async function* saveOps(
         if (files.length === 0) {
             curr = firstAvailableDelta;
         }
-        if (!result.done) {
+        if (result.done === true) {
```

Comment:
Should be `!==` (or explicit check for undefined | false)

Reply:
Oops!